tests/simple.sh

```diff
@@ -0,0 +1,87 @@
+#/bin/bash
+# A quick port of the travis tests to bash
+# TODO evaluate using a framework..
+# We need to be running as root..
+
+PYTHON=${1:-/usr/bin/python3}
+SOSPATH=${2:-./sosreport}
+
+NUMOFFAILURES=0
+
+run_expecting_sucess () {
+FAIL=false
+#Make sure clean
+rm stderr stdout
```

Comment:
Could we have `rm -f` here, please? To prevent potential prompts on RHEL where `rm` is an alias to `'rm -i'`.